.devcontainer/Dockerfile

```diff
@@ -30,8 +29,8 @@ RUN apt-get update && bash /tmp/library-scripts/node-debian.sh "${NVM_DIR}"
 
 RUN pip install starknet-devnet==v0.5.0a2 
 
-# Install protostar
+# Install protostar for vscode user
+USER vscode
 RUN curl -L https://raw.githubusercontent.com/software-mansion/protostar/master/install.sh | bash
```

Comment:
lets remove protostar for now

Reply:
do you mind installing `dojoup` here?

`curl -L https://install.dojoengine.org | bash`